truck_delivery_atge/db_connector_redis.py

```diff
@@ -0,0 +1,63 @@
+from truck_delivery_atge.db_connector import DBConnector
+import redis
+import json
+import threading
+class DBConnectorRedis(DBConnector):
+
+    _instance = None
+    _lock = threading.Lock()
+
+    _instance = None
+
+    def __init__(self):
+         super(DBConnectorRedis, self).__init__()
+
+    @classmethod
+    def instance(cls):
+        if cls._instance is None:
+            cls._instance = cls.__new__(cls)
+            cls.connection = redis.Redis(host="localhost",
+                                          port=6379)
+        return cls._instance
```

Comment:
Nice work doing this, you can check this out as python have dunder methods that can be overwritten to achieve the same result but in a more python way, as it's mentioned here https://python-patterns.guide/gang-of-four/singleton/#a-more-pythonic-implementation